static_src/tests.bundle.js

```diff
@@ -1,3 +1,3 @@
-var context = require.context('./test', true, /.+\.spec\.jsx?$/);
+var context = require.context('./test/unit', true, /.+\.spec\.jsx?$/);
```

Comment:
I'm confused why this matters, since I would've hoped that the local command failed as well.